app/src/main/res/values-da/strings.xml

```diff
@@ -36,11 +40,38 @@
     <string name="show_dialpad">Vis opkaldstastatur</string>
     <string name="hide_dialpad">Skjul opkaldstastatur</string>
     <string name="end_call">Afslut opkald</string>
+    <string name="hold_call">Parker opkald</string>
+    <string name="resume_call">Forsæt opkald</string>
+    <string name="call_on_hold">Opkald parkeret</string>
+    <string name="call_swap">Ombyt opkald</string>
+    <string name="call_merge">Sammenflet opkald</string>
+    <string name="call_split">Opsplit opkald</string>
+    <string name="call_add">Tilføj opkald</string>
+    <string name="conference_manage">Administrer konferenceopkald</string>
+    <string name="conference">Konference</string>
+    <string name="audio_route_speaker">Højtaler</string>
+    <string name="audio_route_earpiece">Øretelefon</string>
+    <string name="audio_route_bluetooth">Bluetooth</string>
+    <string name="audio_route_wired_headset">Kablet hovedtelefon</string>
+    <string name="audio_route_wired_or_earpiece">Kablet øretelefon</string>
+    <string name="choose_audio_route">Vælg audio rute</string>
+    <string name="calling_blocked_number">Nummeret du kalder er blokeret</string>
     <string name="speed_dial">Hurtigopkald</string>
     <string name="manage_speed_dial">Administrér hurtigopkald</string>
     <string name="speed_dial_label">Klik på et nummer for at tilknytte en kontakt til det. Derefter kan du hurtigt ringe op til den givne kontkakt ved at trykke længe på det givne nummer via det numeriske tastatur.</string>
     <string name="group_subsequent_calls">Gruppér efterfølgende opkald med samme nummer i opkaldsloggen</string>
     <string name="open_dialpad_by_default">Åben det numeriske tastatur som standard når appen åbner</string>
     <string name="disable_proximity_sensor">Deaktivér nærhedssensor under opkald</string>
     <string name="disable_swipe_to_answer">Erstat swipe ved besvarelse af indgående opkald med at klikke på</string>
+    <string name="show_incoming_calls_full_screen">Vis altid indkomne opkald i fuld skærm</string>
+    <string name="hide_dialpad_numbers">Skjul numre på tastaturet</string>
+    <string name="export_call_history">Eksporter opkaldshistorik</string>
+    <string name="import_call_history">Importer opkaldshistorik</string>
+    <string name="faq_1_title">Jeg hører indgående opkald, men skærmen tænder ikke. Hvad kan jeg gøre?</string>
+    <string name="faq_1_text">Sådanne problemer kan have mange enheds- og systemspecifikke årsager, så det er svært at sige noget generelt. Du bør se dig omkring i dine enhedsindstillinger og sikre dig, at appen får lov til at poppe op, når den er i baggrunden, og tillade visning over andre apps.</string>
+    <string name="app_launcher_name">Phone_debug</string>
+    <string name="package_name">org.fossify.phone</string>
+    <string name="app_launcher_name">Phone_debug</string>
+    <string name="app_launcher_name">Phone_debug</string>
+    <string name="app_name">Fossify Opkald</string>
```

Comment:
1. Please don't translate `package_name` and `app_name`. They are from donottranslate.xml.
2. You've duplicated `app_launcher_name` entry three times here. Also, it was already translated at the top of the file. Please remove it.

Reply:
I will stop translating package_name, but are you sure you don't want me to translate app_name? 
I don't know why app_launcher_name appear three time. Will delete it.

Reply:
Please take a look:
- https://github.com/orgs/FossifyOrg/discussions/23#discussioncomment-7769471
- https://github.com/orgs/FossifyOrg/discussions/3#discussioncomment-7923864